src/app/pages/home/home.component.html

```diff
@@ -127,7 +127,7 @@ <h3 class="mb-0">Active Requests</h3>
             <tbody *ngFor="let request of requests">
               <tr [ngClass]="{'placed': request.status_name == 'PLACED', 'picked': request.status_name == 'PICKED'}">
                 <th scope="row">
-                  <a [routerLink]="['/user-profile']">{{request.first_name}} {{request.last_name}}</a>
+                  <a [routerLink]="['/user-profile']">{{request.first_name | titlecase}} {{request.last_name | titlecase}}</a>
```

Comment:
this will make the words capitalized, eventhough it is good. it is not a part of this story. There is another ticket that is has taken care of this task.